frontend/src/routes/auth/login/+page.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+	// State variables
+	let showPassword: Boolean = false;
+
+	// Buttons array with common styles
+	let buttons = [
+		{
+			className: 'w-full block bg-black text-white border-2 border-black py-3 text-fluid-md mb-4',
+			content: 'Login',
+			buttonType: 'submit' as const
+		},
+		{
+			className: 'w-full block bg-white border-2 border-black text-black py-3 text-fluid-md mb-4',
+			content: 'Login with Google',
+			buttonType: 'button' as const
+		}
+	];
+</script>
+
+<div class="w-full max-w-md space-y-8">
+	<!-- Header -->
+	<div>
+		<h1 class="mt-6 text-center text-5xl font-extrabold">Log In</h1>
+		<h6 class="text-center mt-6 text-fluid-md">Lorem ipsum dolor sit amet adipiscing elit.</h6>
+	</div>
+
+	<!-- Form -->
+	<form action="/login" method="POST" class="mt-8 space-y-6 flex flex-wrap w-full">
+		<!-- Email input -->
+		<div class="w-full px-3">
+			<label for="email" class="block uppercase tracking-wide text-gray-700 text-sm font-bold"
+				>*EMAIL</label
+			>
+			<input id="email" type="text" class="block w-full bg-white text-gray-700 border py-3 px-3" />
+		</div>
+
+		<!-- Password input -->
+		<div class="w-full px-3">
+			<label for="pw" class="block uppercase tracking-wide text-gray-700 text-sm font-bold"
+				>*PASSWORD</label
+			>
+			<input
+				id="pw"
+				type={showPassword ? 'text' : 'password'}
+				class="block w-full bg-white text-gray-700 border py-3 px-3"
+			/>
+			<button
+				type="button"
+				class="text-fluid-sm text-gray-400 hover:text-gray-900"
+				on:click={() => (showPassword = !showPassword)}
+			>
+				{#if showPassword}
+					Hide Password
+				{:else}
+					Show Password
+				{/if}
+			</button>
+		</div>
+
+		<!-- Buttons -->
+		<div class="w-[95%] mx-auto">
```

Comment:
Avoid arbitary values. Resort to tailwind presets. In this situation, `w-11/12` might be a good choice with a tradeoff of a little bit of width.

OR

You can extend the tailwind config with an additional width preset i.e. `w-11.5/12` which equals to `95%`.